stories/SimplePopup.js

```diff
@@ -17,14 +17,43 @@ const DefaultSimplePopup = () => {
       buttonClass: 'cancel-button',
       dataQeId: 'button-cancel',
     },
+  ];
+
+  return (
+    <Fragment>
+      <button type="button" onClick={() => setIsOpen(true)}>
+        Open SimplePopup
+      </button>
+      <SimplePopup
+        isOpen={isOpen}
+        title="Test Title"
+        close={() => {
+          action()('Close Action Called');
+          setIsOpen(false);
+        }}
+        renderLoader={() => <div className="popup-loader">Loading...</div>}
+        renderContent={() => (
+          <div className="popup-content">some div with content</div>
+        )}
+        options={testOptions}
+      />
+    </Fragment>
+  );
+};
+
+storiesOf('SimplePopup', module).add('default', () => <DefaultSimplePopup />);
+
+const CantCloseClickOutsideModal = () => {
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
looks like we are re-defining a lot, can we not just pass in different props, something like this:
```javascript

const defaultProps = {
  ready: false,
  waiting: false,
  notReadyOrWaitingText: 'failed',
  loadingRenderer: () => <div id="until-ready-loading">loading</div>,
};

const setup = (props = {}) => {
  const testProps = { ...defaultProps, ...props };
  return (
    <UntilReady {...testProps}>
      <div id="until-ready-child">Child content 1</div>
    </UntilReady>
  );
};
```

Reply:
refactored